x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/components/rules_table/add_prebuilt_rules_table/add_prebuilt_rules_table_context.tsx

```diff
@@ -112,6 +119,10 @@ export const AddPrebuiltRulesTableContextProvider = ({
   const { data: prebuiltRulesStatus } = useFetchPrebuiltRulesStatusQuery();
 
   const isUpgradingSecurityPackages = useIsUpgradingSecurityPackages();
+  const noOfPendingInstallAllMutations = useIsMutating({
+    mutationKey: PERFORM_ALL_RULES_INSTALLATION_KEY,
+  });
+  const isInstallAllRulesMutationPending = noOfPendingInstallAllMutations > 0;
```

Comment:
Since the number of active "Install all" mutations can be only 0 or 1, and we are not really interested in the number, I think we can simplify it to a single line.

```suggestion
const isInstallingAllRules =
  useIsMutating({
    mutationKey: PERFORM_ALL_RULES_INSTALLATION_KEY,
  }) > 0;
```

Reply:
Ok, I will improve it. I thought that it's better to be explicit that we first get the number and then convert it to boolean. But I agree now, if it is always 0 or 1, it's better to improve it in the way you proposed.